### ISSUE TITLE
Allow configuring multiple relayhosts

### ISSUE THREAD

**Issue by pirate** [Mailu/Mailu#1097]
I'm trying to setup postfix `sender_dependent_default_transport_maps` to relay through different SMTP servers depending on the sender domain, but I hit a roadblock with adding the sasl passwords.  Currently mailu only supports one relayhost, but I want to add 5-10+.

I propose exposing `/etc/postfix/sasl_passwd` in `/overrides` to let users add their own credentials as a quick fix.

**Comment by kaiyou** [comment]
Agreed with the quick fix, and more generally we could take advantage of that map to use specific relays.

**Comment by pirate** [comment]
For anyone finding this issue trying to do the same thing, this is currently already doable by specifying your own `sasl_passwd` file in `overrides/postfix.cf`, you can see an example here: https://github.com/Mailu/Mailu/issues/1096

The enhancement would be to automatically use any `overrides/sasl_passwd` file present without having to add the line to `overrides/postfix.cf` yourself. :+1:

**Comment by Nebukadneza** [comment]
Hi There,

The `Mailu`-Project is currently in a bit of a bind! We are short on man-power, and we need to judge if it is possible for us to put in some work on this issue.

To help with that, we are currently trying to find out which issues are actively keeping users from using `Mailu`, which issues have someone who want to work on them — and which issues may be less important. These a less important ones could be discarded for the time being, until the project is in a more stable and regular state once again.

In order for us to better assess this, it would be helpful if you could put a **reaction on this post** (use the :smiley: icon to the top-right).
- 👍️ if you **need this** to be able to use Mailu. Ideally, you’d also be able to test this on your installation, and provide feedback …
- 🎉 if you find it a **nice bonus**, but no deal-breaker
- 🚀 if you want to **work on it yourself**!
We want to keep this **voting open for 2 weeks** from now, so please help out!

**Comment by pirate** [comment]
boop

**Comment by Omrigan** [comment]
I'd like to have an ability to send my emails from e.g. my @gmail.com address so that I don't have to configure different SMTP credentials in all my clients. Is this a correct place to discuss such an enhancement?
